documentation/src/main/docs/faq.adoc

```diff
@@ -572,6 +572,19 @@ It gets the items from the lists and passes them downstream:
 include::plantuml/disjoint.puml[]
 ----
 
+`Multi.createBy().repeating()` lets you should the number of iteration using:
+
+* `atMost` - exact number of repetition (or a failure happens)
+* `until` - the repetition is stopped if the item emitted by the `Uni` passed a test (predicate). The failing item is not propagated downstream.
```

Comment:
This seems contradictory as you say the repetition is stopped if the item passes a test, but then refer to a failing item

Reply:
failing is not right, it's the test that didn't pass the test (it's not a failure). Let me rewrite.

Reply:
Ok, I've rewritten the section - let me know.